src/v/redpanda/admin/api-doc/partition.json

```diff
@@ -299,6 +299,21 @@
                 }
             ]
         },
+        {
+            "path": "/v1/partitions/rebalance_cores",
+            "operations": [
+                {
+                    "method": "POST",
+                    "summary": "Trigger core placement rebalancing for partitions in this node.",
```

Comment:
```suggestion
                    "summary": "Trigger core placement rebalancing for partitions in this broker.",
```
Same question here about "shard" versus "core", and wanted to double check usage of "node" instead of "broker" -- in docs we have defaulted to "broker"